addons/delivery/views/delivery_view.xml

```diff
@@ -295,7 +281,7 @@
             <field name="inherit_id" ref="stock.view_quant_package_tree"/>
             <field name="arch" type="xml">
                 <field name="display_name" position="after">
-                    <field name="packaging_id"/>
+                    <field name="package_type_id"/>
```

Comment:
I would put the field in stock module as you did for the form view (unless you already discussed that with others)